src/actors/rqgActorSheet.ts

```diff
@@ -738,6 +739,54 @@ export class RqgActorSheet extends ActorSheet<
       });
     });
 
+    // Roll Rune Magic
+    this.form?.querySelectorAll("[data-rune-magic-roll]").forEach((el) => {
```

Comment:
add `[data-rune-magic-roll]`to rqg.scss around line 562 to get a hover effect

Reply:
Done